dev/TEAL_opcodes.md

```diff
@@ -618,6 +618,27 @@ See `bnz` for details on how branches work. `b` always jumps to the offset.
 - immediately fail unless A is a non-zero number
 - Availability: v3
 
+## bury n
+
+- Opcode: 0x45 {uint8 depth}
+- Stack: ..., A &rarr; ...
+- Replace the Nth value from the top of the stack. bury 0 fails.
+- Availability: v8
+
+## popn n
+
+- Opcode: 0x46 {uint8 stack depth}
+- Stack: ..., [N items] &rarr; ...
+- Remove N values from the top of the stack
```

Comment:
What happens if less than N values in the stack

Reply:
panic